### PR TITLE
Added new library app project

### PR DESCRIPTION
**Pull request by Muskan-creator**
## Related Issue
  Added a new library app project in which  user can make a list of books with their authors, book pages and keep them as records. You just have to type the book information in the text fields and click on the add button to add the information on the record. This project includes a lot of javascript for making the functioning of the project. The user can add the number of book details and you can see the details stored in the list form. The user can delete the list of items if he/she wants to remove them.

Closes: #1050 

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

### Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

### Screenshots
Put any screenshot(s) of the project here.
image

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -63,6 +63,7 @@ a{
   outline: none;
```
Comment (991rajat) [inline assets/css/styles.css:63]: this file is not required I think. Don't push this one.

**Review by 991rajat — CHANGES_REQUESTED** [review]
Works great. Do you know CSS? If not tell me, will create a new issue for styling
Rightnow it looks pretty simple, add some css to make it look good.

And also reduce the size of the gif if possible. If not you can create a staic image and replace it with. This will be of smaller size.

**Comment by Muskan-creator** [comment]
@991rajat I have added more styling to project and also resolve conflicts.
Please merge it and also add levels to it .

File: script.js [hunk 2]
```diff
@@ -51,11 +51,22 @@ let projectData = [
...
+    projectName: "Library-App",
+    projectImage: "assets/GIFs/libraryapp.gif",
+    projectUrl: "public/libraryapp.html",
+  },
+  {
+    projectName: "Counter",
+    projectImage: "assets/GIFs/counter.gif",
+    projectUrl: "public/counter.html",
+  },
     projectName: 'JavaScript Interest Calculator',
```
Comment (991rajat) [inline script.js:64]: Remove the counter and there is { missing for interest calculator.

**Review by 991rajat — CHANGES_REQUESTED** [review]
Looks Good, Just do the changes in script.js. It will be ready to merge.

**Pull request merged**